<launch>
  <node pkg="multirobot_detect_iarc" name="multirobot_detect_iarc_node" type="multirobot_detect_iarc_node" output="screen">
  <!--param name="subscribed_topic" type="string" value="/dji_sdk/image_raw"/-->
  <param name="subscribed_topic" type="string" value="/my_video"/>
  <param name="show_video_flag" type="bool" value="true"/>
  <param name="save_result_video_flag" type="bool" value="true"/>
  <param name="result_video_file_name" type="string" value="/home/zj/ros_workspace/src/multirobot_detect_iarc/result/raw.avi"/>
  <param name="rate" type="double" value="5.0"/>
  
  <param name="camera_pitch" type="double" value="-36.0"/>
  <param name="fu" type="double" value="376.629954"/>
  <param name="fv" type="double" value="494.151786"/>
  <param name="listen_h_flag" type="bool" value="false"/>
  <param name="loc_h" type="double" value="1.6"/>
  </node>
</launch>
